paragraphs.module

```diff
@@ -87,6 +87,19 @@ function paragraphs_ctools_plugin_directory($module, $plugin) {
   }
 }
 
+/**
+ * Implements hook_config_info().
+ */
+function paragraphs_config_info() {
+  return array(
+    'paragraphs.type' => array(
+      'name_key' => 'bundle',
+      'label_key' => 'name',
+      'group' => t('Paragraphs types'),
```

Comment:
Other places in the UI we use `Paragraph types` -- can you update to follow that pattern? (I realize the module name contains the extra `s` but something about `Paragraphs types` just feels like a mouthful of marbles)

Reply:
No problem, done.